src/modules/peek/Peek.FilePreviewer/UnsupportedFilePreview.xaml.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Peek.FilePreviewer
+{
+    using CommunityToolkit.Mvvm.ComponentModel;
+    using Microsoft.UI.Xaml.Controls;
+    using Microsoft.UI.Xaml.Media.Imaging;
+
+    [INotifyPropertyChanged]
+    public sealed partial class UnsupportedFilePreview : UserControl
+    {
+        [ObservableProperty]
```

Comment:
I know this works fine and is much less code, but usually I'd go for dependency properties on a usercontrol.